apple/internal/aspects/swift_dynamic_framework_aspect.bzl

```diff
@@ -133,7 +133,7 @@ single swift_library dependency.\
             generated_header = headers.pop(0)
 
     # Make sure that all dictionaries contain at least one module before returning the provider.
-    if all([module_name, generated_header, swiftdocs, swiftmodules, modulemap_file]):
```

Comment:
Doesn't the modulemap not end up existing in this case too?

Reply:
It still existed in for us.

Reply:
Ahh, it shouldn't be created. I'll update that.